tasks.py

```diff
@@ -2,9 +2,9 @@
 import inspect
 import operator
 import os
-import re
-import pkg_resources
-import platform
+import toml
```

Comment:
use [tomli](https://pypi.org/project/tomli/), it has been more recently updated
```suggestion
import tomli
```